### PR TITLE
Pull Request

### PR DESCRIPTION
**Pull request by thanaphat707**

### CODE REVIEW

File: tests.cpp [hunk 1]
```diff
@@ -14,9 +14,12 @@ TEST(SquareRootTest, PositiveNos) {
 void hello()
 {
     int a;
+    printf("sasdadad");   // comment
```
Comment (makomawin) [inline tests.cpp:17]: Please  change.to  support tanathorn

**Review by makomawin — CHANGES_REQUESTED** [review]

File: tests.cpp [hunk 2]
```diff
@@ -14,9 +14,12 @@ TEST(SquareRootTest, PositiveNos) {
 void hello()
 {
     int a;
```
Comment (memo-jun) [inline tests.cpp]: Plz, remove this line

**Review by thanaphat707 — COMMENTED** [review]
Revise

**Review by memo-jun — CHANGES_REQUESTED** [review]

**Review by makomawin — APPROVED** [review]
Sathuu

**Review by memo-jun — APPROVED** [review]
LGTM🤔

**Pull request merged**